### ISSUE TITLE
Tests for WCAG technique G192: Fully conforming to specifications

### ISSUE THREAD

**Issue by jessebeach** [quailjs/quail#89]
This issue will most likely result in several tests. Please open more issues and link them as appropriate.

Complete automated testing is likely not possible to satisfy this technique. The best we can do is give the user a sense of how much human attention might be required so that their evaluation queue can be prioritized.

Most likely we will want to use a third-party library for this issue. This library will need to be MIT license compatible.
## Testing

From http://www.w3.org/TR/WCAG20-TECHS/G192.html
### Procedure

Check that all technologies are used according to specification.

Note: While validators can be great tools for catching errors, they usually cannot catch all cases where content fails to fully conform to a specification.
### Expected Results

Check #1 is true.

If this is a sufficient technique for a success criterion, failing this test procedure does not necessarily mean that the success criterion has not been satisfied in some other way, only that this technique has not been successfully implemented and can not be used to claim conformance.
